internal/stacks/stack.go

```diff
@@ -62,21 +72,22 @@ func ListStacks() ([]string, error) {
 	return stacks, nil
 }
 
-func InitStack(stackName string, memberCount int) error {
+func InitStack(stackName string, memberCount int, options *InitOptions) error {
```

Comment:
I wonder if `stackName` and `memberCount` should also be part of `*InitOptions`? I'm trying to think if there's a reason to think of them as a separate thing. Are they not also options for initialization? Or should we make a distinction between required and optional arguments this way?

Reply:
Did you want me to make that change in this PR, or leave that with you for a future enhancement?

Reply:
I'm fine either way. Just thinking of how we can try to improve the code as we go. I'm okay with the implementation as-is though.

Reply:
Merging this one, so I can use it in the e2e test (which updates the CLI from go get during its operation)